### PR TITLE
New themes

### PR DESCRIPTION
**Pull request by Rainb0wKey**

### CODE REVIEW

**Comment by Rainb0wKey** [comment]
Test

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]
Hello Rainb0wKey,
Do update `book.js` and `index.hbs` too. Your PR is incomplete!

**Comment by Rainb0wKey** [comment]
Ready!

File: src/theme/index.hbs [hunk 1]
```diff
@@ -117,6 +117,8 @@
                             <ul id="theme-list" class="theme-popup" aria-label="Themes" role="menu">
                                 <li role="none"><button role="menuitem" class="theme" id="bdfd">{{ theme_option "Bot Designer for Discord" }}</button></li>
                                 <li role="none"><button role="menuitem" class="theme" id="shiro">{{ theme_option "Shiro's Theme" }}</button></li>
+				<li role="none"><button role="menuitem" class="theme" id="green">{{ theme_option "Green" }}</button></li>
+				<li role="none"><button role="menuitem" class="theme" id="ios">{{ theme_option "IOS theme" }}</button></li>
```
Comment (priyanujgogoi-28) [inline src/theme/index.hbs]:
```suggestion
				<li role="none"><button role="menuitem" class="theme" id="ios">{{ theme_option "iOS" }}</button></li>
```

File: src/theme/index.hbs [hunk 2]
```diff
@@ -124,6 +124,8 @@
                                 <li role="none"><button role="menuitem" class="theme" id="coal">{{ theme_option "Coal" }}</button></li>
                                 <li role="none"><button role="menuitem" class="theme" id="navy">{{ theme_option "Navy" }}</button></li>
                                 <li role="none"><button role="menuitem" class="theme" id="ayu">{{ theme_option "Ayu" }}</button></li>
+				<li role="none"><button role="menuitem" class="theme" id="green">{{ theme_option "Green" }}</button></li>
+				<li role="none"><button role="menuitem" class="theme" id="ios">{{ theme_option "iOS" }}</button></li>
```
Comment (noituri) [inline src/theme/index.hbs]: Please indent the code
```suggestion
                                <li role="none"><button role="menuitem" class="theme" id="coal">{{ theme_option "Coal" }}</button></li>
                                <li role="none"><button role="menuitem" class="theme" id="navy">{{ theme_option "Navy" }}</button></li>
                                <li role="none"><button role="menuitem" class="theme" id="ayu">{{ theme_option "Ayu" }}</button></li>
                                <li role="none"><button role="menuitem" class="theme" id="green">{{ theme_option "Green" }}</button></li>
                                <li role="none"><button role="menuitem" class="theme" id="ios">{{ theme_option "iOS" }}</button></li>
```

**Review by noituri — CHANGES_REQUESTED** [review]

**Review by priyanujgogoi-28 — APPROVED** [review]
Thank you for contributing!

**Comment by Rainb0wKey** [comment]
### Thanks!

**Review by MineBartekSA — APPROVED** [review]

**Review by noituri — APPROVED** [review]

**Pull request merged**